test/ExportQueryExecutionTreesTest.cpp

```diff
@@ -1078,8 +1099,9 @@ TEST_P(StreamableMediaTypesFixture, CancellationCancelsStream) {
   auto qet = qp.createExecutionTree(pq);
 
   cancellationHandle->cancel(ad_utility::CancellationState::MANUAL);
+  ad_utility::Timer timer(ad_utility::Timer::Started);
   auto generator = ExportQueryExecutionTrees::computeResultAsStream(
-      pq, qet, GetParam(), std::move(cancellationHandle));
+      pq, qet, GetParam(), timer, std::move(cancellationHandle));
```

Comment:
This test suite now should also cover the two new `JSON` exporters, so just add them to the
` INSTANTIATE_TESt_SUITE_P` in line 1110 below.